lib/topics/state_and_behaviour.rb

```diff
@@ -1,5 +1,27 @@
 # frozen_string_literal: true
 
-# exercise state and behaviour
 module StateAndBehaviour
+  # describe cars
+  class Car
+    attr_accessor(:year, :color, :model, :current_speed)
```

Comment:
please read [parentheses section](https://github.com/bbatsov/ruby-style-guide#method-invocation-parens) in style guide